### PR TITLE
Add the section Mallet deploy contract

### PR DESCRIPTION
**Pull request by johnny7861532**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ CNS is the Cardano Name Service, a distributed, open, and extensible naming syst
 
 #### Table of Contents
 -  [Introduction](./docs/INTRODUCTION.md)
+- [Using Mallet To Deploy Contract](https://github.com/johnny7861532/cns/blob/master/depoly_md/Cardano%20install%20Mallet%20to%20deploy%20and%20interact%20with%20smart%20contract.md)
```
Comment (PhyrexTsai) [inline README.md]: You can use relative path of the file, just like (./docs/INTRODUCTION.md)

File: depoly_md/Cardano install Mallet to deploy and interact with smart contract.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+         -d, --datadir      Specify data directory      optional      default: "/Users/johnnyhsieh/.mallet2"
+    
+       GLOBAL OPTIONS
+    
+         -h, --help         Display help
+         -V, --version      Display version
+         --no-color         Disable colors
+         --quiet            Quiet mode - only displays warn and error messages
+         -v, --verbose      Verbose mode - will also output debug messages
+Before we can start to interact with Cardano
```
Comment (PhyrexTsai) [inline depoly_md/Cardano install Mallet to deploy and interact with smart contract.md:56]: You can use ``` wrap this section

File: depoly_md/Cardano install Mallet to deploy and interact with smart contract.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+mallet> requestFunds()
+    '0x27e2e4d47d5474f599fa9a676d819e39e36a5684c550ddce28606ef6035ef01c'
+
+mallet> getBalance()
+    '30000000000000000'
+```
+
+
+
+You cna also check the transaciton receipt with `$getReceipt`
```
Comment (PhyrexTsai) [inline depoly_md/Cardano install Mallet to deploy and interact with smart contract.md]: You can

File: depoly_md/Cardano install Mallet to deploy and interact with smart contract.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    returnData: '0xd49348656c6c6f20576f726c64000000000000000b' }
+```
+
+
+
+And the contract return "0xd49348656c6c6f20576f726c64000000000000000b", let translate into string
+
+
+
+![螢幕快照 2018-09-21 下午3.56.42](https://github.com/johnny7861532/cns/blob/master/depoly_md/translate.png)
```
Comment (PhyrexTsai) [inline depoly_md/Cardano install Mallet to deploy and interact with smart contract.md]: Please user the relative path for this image

**Review by PhyrexTsai — CHANGES_REQUESTED** [review]

File: depoly_md/Cardano install Mallet to deploy and interact with smart contract.md (hunk 3 above)
Comment (johnny7861532) [reply depoly_md/Cardano install Mallet to deploy and interact with smart contract.md]: don't get  what do u mean this line of require.

**Review by PhyrexTsai — APPROVED** [review]

**Pull request merged**